### PR TITLE
#6: Adding contributors guide

### PR DESCRIPTION
**Pull request by jammie1903**
Contributors guide first draft
First time writing one, hope its up to scratch
#6

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Contributing
+
+If you wish to make a contribution to this project, please make sure there is a relevant issue created for it first. Also, please assign (or ask a contributor to assign) the relevant issue to yourself whislt working on it, to avoid any wasted effort by another developer.
```
Comment (joshghent) [inline docs/CONTRIBUTING.md]: Correct spelling of "whilst"

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- The release process is unfortunantly heavily tied to intellij, as it relies on the "artifacts" created by the IDE's build process. If you do not use intellij, or do not feel confident performing a release, you can request a release from me (jammie1903).
+
+- A release will be necessary for any users of the software to see your changes, merging to master is not enough!
+
+### Before the release
+- Ensure the version file is updated relevantly and this change is committed `src/resources/version.txt`
+- Make sure the artifact has been successfully built. You can run it for testing purposes straight out of the artifacts folder.
+
+### Preforming a release
+project-builder has its own built in release script, found in `/src/main/com/jamie/releaser/Release.java`, simply run this, and it will release the built artifact to the repo.
```
Comment (joshghent) [inline docs/CONTRIBUTING.md:21]: Does the release.java also do the git tagging and stuff? If so, neat! If not, make a note to do this work as well.

**Review by joshghent — APPROVED** [review]
Looks good after the comments!

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (jammie1903) [reply docs/CONTRIBUTING.md:21]: the git release process creates a tag for you, so that's taken care of :+1:

Comment (joshghent) [reply docs/CONTRIBUTING.md:21]: That's sick mate :fire:

**Pull request merged**